app/controllers/restful_redirect_controller.rb

```diff
@@ -6,9 +6,7 @@ def index
     when 'ExtManagementSystem'
       record = ExtManagementSystem.find_by(:id => params[:id])
       if record
-        if %w[ManageIQ::Providers::ConfigurationManager].include?(record.type) || record.type.starts_with?('ManageIQ::Providers::Foreman')
-          redirect_to(:controller => 'provider_foreman', :action => 'show', :id => params[:id])
-        elsif %w[ManageIQ::Providers::AnsibleTower::AutomationManager].include?(record.type)
+        if %w[ManageIQ::Providers::AnsibleTower::AutomationManager].include?(record.type)
           redirect_to(:controller => 'automation_manager', :action => 'show', :id => params[:id])
         elsif %w[ManageIQ::Providers::EmbeddedAnsible::AutomationManager].include?(record.type)
```

Comment:
Same here - `if record.kind_of?(ManageIQ::Providers::EmbeddedAutomationManager)`

Reply:
I see technically this was original code (didn't realize initially), so this can be a followup